glmtools/tests/test_glm.py

```diff
@@ -16,4 +17,48 @@
 def test_glm_t_test():
     # Test glm and t_test against scipy
     # Your test code here
+
+    #creating an independent set of random numbers to use for testing
```

Comment:
Also - just FYI - the estimation of the betas does not depend on these values being independent.  The t test and p value do depend on the values being independent.